### PR TITLE
DTGB-659: Reverted tablet breakpoint to 768px

### PR DESCRIPTION
**Pull request by hstandaert**
## Description

- Changed tablet breakpoint back to `768px`
- Optimized styling for some components: 
  - Footer
  - Filter layout

## Motivation and Context

Tablet breakpoint was `769px`, which doesn't make sense because iPad portrait (for example) is not included in this range.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [x] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: styleguide/components/61-layouts/sidebar-layout/_sidebar-layout.scss [hunk 1]
```diff
@@ -8,22 +8,23 @@
...
     }
   }
 
   .content {
     width: 100%;
 
-    @include tablet {
-      width: span(8 wide of 12);
-    }
+    //@include tablet {
```
Comment (meirege) [inline styleguide/components/61-layouts/sidebar-layout/_sidebar-layout.scss]: Remove commented code.

File: styleguide/components/61-layouts/sidebar-layout/_sidebar-layout.scss [hunk 2]
```diff
@@ -8,22 +8,23 @@
   .sidebar {
     width: 100%;
 
-    @include tablet {
-      width: span(4 of 12);
-      padding-right: gutter();
-    }
+    //@include tablet {
```
Comment (meirege) [inline styleguide/components/61-layouts/sidebar-layout/_sidebar-layout.scss]: Remove commented code.

File: styleguide/components/41-organisms/footer/_footer.scss [hunk 3]
```diff
@@ -136,9 +136,9 @@
     justify-content: center;
     width: 100%;
 
-    @include tablet {
-      justify-content: start;
-    }
+    //@include tablet {
```
Comment (meirege) [inline styleguide/components/41-organisms/footer/_footer.scss]: Remove commented code.

File: styleguide/components/41-organisms/footer/_footer.scss [hunk 4]
```diff
@@ -100,11 +100,11 @@
           margin-bottom: 0;
         }
 
-        @include tablet {
-          order: 2;
-          width: span(8 of 12);
-          margin-bottom: 1.8rem;
-        }
+        //@include tablet {
```
Comment (meirege) [inline styleguide/components/41-organisms/footer/_footer.scss]: Remove commented code.

**Review by meirege — CHANGES_REQUESTED** [review]

**Review by delrueba — APPROVED** [review]

**Review by meirege — APPROVED** [review]

**Pull request merged**